src/aleph/services/cost.py

```diff
@@ -12,9 +13,8 @@
 
 
 def _get_file_from_ref(
-    session: DbSession, ref: str, use_latest: bool
+        session: DbSession, ref: str, use_latest: bool
```

Comment:
Same, please format your PRs with `black`. It seems like you have a different formatter running automatically.